lib/tmdb/client.rb

```diff
@@ -21,21 +21,53 @@ def movie_search(movie_title)
         )
       end
 
-      def movie(movie_id)
-        data = get_parsed_movie_data(movie_id)
-        MovieMore.parse_result(data)
+      def get_movies_for_actor(actor_name:, page:, sort_by:)
+        person_url = url_for_person_search(actor_name)
+        person_data = get_data(person_url)&.dig(:results)&.first
+        if person_data.blank?
+          OpenStruct.new(not_found_message: "No person matches found for '#{actor_name}'.")
+        else
+          movie_url = url_for_movie_discover_search(
+            people: person_data[:id],
+            page: page,
+            sort_by: sort_by
+          )
+          movie_data = get_data(movie_url)
+          movie_results = movie_data&.dig(:results)
```

Comment:
Do we need the separate `movie_data` variable? It doesn't look like it's used anywhere else.

Reply:
It's used on line 37. It's annoying, but we need the pagination info and it's at the same level as the `:results` key:
```
movie_data = get_data(movie_url)
movie_results = movie_data&.dig(:results) <--
total_pages = movie_data&.dig(:total_pages) <--
```